_pkgdown.yml

```diff
@@ -1,3 +1,29 @@
 url: https://viralemergence.github.io/wddsWizard/
 template:
   bootstrap: 5
+reference:
+- title: Data
+  desc: Example datasets and other useful objects
+- contents:
+  - has_concept("Data")
+- title: Templates
+  desc: Templates for creating data that conforms to the standard
+- contents:
+  - has_concept("Templates")
+- title: WDDS Deposit
+  desc: Functions for interacting with the Zenodo Deposit
```

Comment:
For all `desc` I think you can simplify by removing "functions", "templates", etc, which is implied by the context